src/conn/conn_compact.c

```diff
@@ -213,6 +213,10 @@ __background_compact_should_run(WT_SESSION_IMPL *session, const char *uri, int64
     if (compact_stat == NULL)
         return (true);
 
+    /* If we are running once, force compaction on the file. */
```

Comment:
Based on the `run_once` flag, we force each file to compact irrespective of its stats gathered earlier. I felt we shouldn't change this behaviour. I am fine with force, too, if others agree.

Reply:
Updated the intent of that field in b9c413261c3966f8f656d4e7c218c6caf1869114. We want to force compaction on each table when `run_once` is set.